source/includes/steps-install-bi-connector.yaml

```diff
@@ -23,9 +23,8 @@ action:
 
   - heading: "Windows"
     pre: | 
-
-      Delete the MongoDB BI Connector directory from the :guilabel:`Program
-      Files` directory.
+      Delete the MongoDB BI Connector directory from the 
```

Comment:
I simplified the install page somewhat, removing the explicit uninstall step (since macOS users will overwrite and the Windows users will get a prompt from the Install Wizard) and moving the OpenSSL part to a prerequisite.